script/test

```diff
@@ -26,7 +29,7 @@ if [ $# -eq 0 ]
 then
     printf "\nChecking that tasks build correctly\n\n"
     $COMPOSE shards build
-    
+
     printf "\nChecking code formatting\n\n"
```

Comment:
Whoops, didn't notice this whitespace change. I guess my editor automatically kills whitespace at the end of lines. I can revert this if desired.

Reply:
No this is great! I'm glad your editor caught it :D